content/en/security/application_security/getting_started/serverless.md

```diff
@@ -0,0 +1,147 @@
+---
+title: AWS Lambda Getting Started with ASM
+kind: documentation
+is_beta: true
+code_lang: serverless
+type: multi-code-lang
+code_lang_weight: 90
+further_reading:
+    - link: "/security/application_security/how-appsec-works/"
+      tag: "Documentation"
+      text: "How Application Security Works"
+    - link: "/security/default_rules/#cat-application-security"
+      tag: "Documentation"
+      text: "OOTB Application Security Management Rules"
+    - link: "/security/application_security/troubleshooting"
+      tag: "Documentation"
+      text: "Troubleshooting Application Security Management"
+    - link: "/security/application_security/threats/"
+      tag: "Documentation"
+      text: "Application Threat Monitoring and Protection"
+    - link: "/security/application_security/risk_management/"
+      tag: "Documentation"
+      text: "Application Risk Management"
+---
+
+{{< callout url="#" header="ASM support for AWS Lambda is in beta" btn_hidden="true" >}}
+  Threat detection is done by using the Lambda extension.
+{{< /callout >}}
+
+You can monitor your functions running in AWS Lambda with Datadog Application Security Management (ASM). 
+
+In general, setting up ASM for AWS Lambda involves:
+
+1. Identifying functions that are vulnerable or are under attack, which would most benefit from ASM. Find them on [the Security tab of your Service Catalog][1].
+2. Set up ASM instrumentation via the serverless plugin or manually setting the ARN.
+3. Enabling the library to collect the application security data from the functions and send it to Datadog.
+4. Triggering security signals in your application and seeing how Datadog displays the resulting information.
+
+## Prerequisites
+
+- [Serverless APM][2] is configured to send traces directly to Datadog (can’t use X-Ray integration) 
```

Comment:
Let's make this more explicit

"- [Serverless APM][2] is configured on the lambda function to send traces directly to Datadog (can’t use X-Ray integration)"